### PR TITLE
MRG: update with misc animal genomes

### PR DESCRIPTION
**Pull request by ctb**
Adds common hosts and also hg38.

Tackles https://github.com/sourmash-bio/sourmash/issues/2717

Screenshot 2024-12-07 at 9 17 40 AM

## Rendered preview:

### CODE REVIEW

**Comment by ctb** [comment]
Ready for review & merge @bluegenes @ccbaumler !

File: doc/databases.md [hunk 1]
```diff
@@ -30,6 +30,20 @@ The databases do not need to be unpacked or prepared in any way after download.
 
 You can verify that they've been successfully downloaded (and view database properties such as `ksize` and `scaled`) with `sourmash sig summarize <output>`.
 
+## Sketches for human and animal genomes
+
+These include k=21, k=31, and k=51, at a scaled of 1000. Each file is about 110 MB.
```
Comment (ccbaumler) [inline doc/databases.md]:
```suggestion
These signature files are representative sketches of model organisms. There suggested usage is to identify and remove contamination from environmental sketches. These include k=21, k=31, and k=51, at a scaled of 1000. Each file is about 110 MB.
```

**Comment by ccbaumler** [comment]
Also, would you like to add some of the more canonical organisms? Namely,

1. rat https://www.ncbi.nlm.nih.gov/datasets/taxonomy/10116/
2. xenopus https://www.ncbi.nlm.nih.gov/datasets/taxonomy/8355/
3. zebrafish https://www.ncbi.nlm.nih.gov/datasets/taxonomy/7955/
4. drosophila https://ncbi.nlm.nih.gov/datasets/taxonomy/7227/
5. c. elegans https://www.ncbi.nlm.nih.gov/datasets/taxonomy/6239/

File: doc/databases.md (hunk 1 above)
Comment (ctb) [reply doc/databases.md]: updated in dc9d4353 with:
+These sketches are of the latest releases of a number of animal
+genomes. Among other uses, they can be used to detect host
+contamination in microbial metagenomes.
+
+Each file includes sketches at k=21, k=31, and k=51, at a scaled of
+1000, and is about 110 MB.

**Comment by ctb** [comment]
> Also, would you like to add some of the more canonical organisms? Namely,
> 
>1. rat https://www.ncbi.nlm.nih.gov/datasets/taxonomy/10116/
> 
>2. xenopus https://www.ncbi.nlm.nih.gov/datasets/taxonomy/8355/
> 
>3. zebrafish https://www.ncbi.nlm.nih.gov/datasets/taxonomy/7955/
> 
>4. drosophila https://ncbi.nlm.nih.gov/datasets/taxonomy/7227/
> 
>5. c. elegans https://www.ncbi.nlm.nih.gov/datasets/taxonomy/6239/

No, I just want to merge ;).

I've added these to https://github.com/sourmash-bio/sourmash/issues/2717 so we don't lose the suggestion. But the pipeline I'm using doesn't do anything fancy and I don't want to piecemeal add new genomes (because it's work, and it's also error prone ;) until we automated the process more.

**Review by ccbaumler — APPROVED** [review]
LGTM

**Pull request merged**